airmozilla/main/templates/main/home.html

```diff
@@ -11,7 +11,50 @@
 </style>
 {% endblock %}
 
+{% macro article(event) %}
+{% set href = url('main:event', slug=event.slug) %}
+<article id="event-{{ event.id }}" class="post type-post status-publish format-standard hentry">
+  <header class="entry-header">
+    <h2 class="entry-title">
+      <a href="{{ href }}">{{ event.title }}</a>
+    </h2>
+    {% set curated_groups = get_curated_groups(event) %}
+    {% include 'main/_event_privacy.html' %}
+  </header>
+  <div class="entry-summary">
+    <p class="event-date">{{ event.start_time|js_date }}</p>
+    <a href="{{ href }}">
+      {{ show_thumbnail(event) }}
+    </a>
+    <a class="star star-off"></a>
+    <p>
+      {{ short_desc(event) | safe_html }}
+      <a class="go" href="{{ href }}">{{ _('See more') }}</a>
+    </p>
+  </div>
+</article>
+{% endmacro %}
 
+{% macro event_paginate(next_page_url, tags, prev_page_url) %}
```

Comment:
Make `tags` the last argument so that `next_page_url` and `prev_page_url` can be together. 
Also, since `tags` is something `None` or `''` you might want to write it as `tags=None` in the parameter list. 
I think for the starred events won't ever have tags.